sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryExceptionData.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Azure.Core;
+using OpenTelemetry.Logs;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Models
+{
+    internal partial class TelemetryExceptionData
+    {
+        internal const int MaxExceptionCountToSave = 10;
+
+        public TelemetryExceptionData(int version, LogRecord logRecord) : base(version)
+        {
+            SeverityLevel = LogsHelper.GetSeverityLevel(logRecord.LogLevel);
+            ProblemId = LogsHelper.GetProblemId(logRecord.Exception);
+
+            var message = logRecord.State.ToString();
```

Comment:
`State` could be null

Reply:
Should we look at `logrecord.FormattedMessage` first ( if `IncludeFormattedMessage=true`) and then `logrecord.State` (if `ParseStateValues=false`) and if that also fails look at `logrecord.StateValues` {OriginalFormat} ?